openvm/tests/common/mod.rs

```diff
@@ -60,9 +60,17 @@ pub mod apc_builder_utils {
             start_pc: 0,
         };
 
-        let apc =
-            build::<BabyBearOpenVmApcAdapter>(basic_block.clone(), vm_config, degree_bound, None)
-                .unwrap();
+        // Use this env var to output serialized APCs for tests as well.
+        let apc_path_var = std::env::var("APC_CBOR_PATH").ok();
```

Comment:
won't these generated files from different tests conflict with each other?

Reply:
Yea but the usage idea was that whoever is interested in this would be running one test at a time. But I could also add a prefix arg

Reply:
maybe its worth it?